api/evidence/evidence_store.proto

```diff
@@ -58,6 +58,11 @@ service EvidenceStore {
   rpc GetEvidence(GetEvidenceRequest) returns (Evidence) {
     option (google.api.http) = {get: "/v1/evidence_store/evidences/{evidence_id}"};
   }
+
+  // Returns the resource types that are supported by the EvidenceStore.
+  rpc GetSupportedResourceTypes(GetSupportedResourceTypesRequest) returns (GetSupportedResourceTypesResponse) {
```

Comment:
Because you don't provide any content in `GetSupportedResourceTypesRequest` and you actually list all resource types, I would rename it to ListXXX